README.rst

```diff
@@ -110,30 +117,54 @@ If no specific bucket name is provided, the "default" bucket is used.
 
     client = Client(server_url='http://localhost:8888/v1',
                     auth=credentials)
+
+    # To create a bucket.
     client.create_bucket('payments')
-    client.get_bucket('payments')
+
+    # To get an existing bucket 
+    bucket = client.get_bucket('payments')
+
+    # Or retriave all readable buckets.
+    buckets = client.get_buckets()
+
+    # To update a bucket.
```

Comment:
Both PUT/PATCH do an *update*, we could say *create or replace* for `PUT` and *modify* for `PATCH`? What do you think?